src/kudu/consensus/log.h

```diff
@@ -110,16 +115,17 @@ class Log : public RefCountedThreadSafe<Log> {
                      const scoped_refptr<MetricEntity>& metric_entity,
                      scoped_refptr<Log> *log);
 
-  ~Log();
+  virtual ~Log();
 
   // Synchronously append a new entry to the log.
   // Log does not take ownership of the passed 'entry'.
-  Status Append(LogEntryPB* entry);
+  virtual Status Append(LogEntryPB* entry);
```

Comment:
this function will not be used on mysql side. Do we need it virtual?

Reply:
it would be nice to put asserts that these functions never get called when the log abstraction is overriden by mysql